src/components/profile/ProfileDataTable.js

```diff
@@ -40,87 +39,85 @@ const ProfileDataTable = ({
+      <KeyboardAwareScrollView resetScrollToCoords={{ x: 0, y: 0 }} scrollEnabled={false}>
+        <Section.Row>
+          <InputRounded
+            disabled={!editable}
+            error={errors.username}
+            icon="username"
+            iconColor={theme.colors.primary}
+            iconSize={22}
+            onChange={username => onChange({ ...profile, username })}
+            placeholder="Choose a Username"
+            value={profile.username}
+          />
+        </Section.Row>
+        <Section.Row>
+          {editable ? (
+            <Section.Stack grow>
+              <Section.Row>
+                <EditPhoneInput
+                  error={errors.mobile && errors.mobile !== ''}
+                  id="signup_phone"
+                  onFocus={() => setLockSubmit(true)}
+                  onChange={value => onChange({ ...profile, mobile: value })}
+                  onBlur={() => {
+                    setLockSubmit(false)
+                    verifyPhone()
+                  }}
+                  placeholder="Enter phone number"
+                  value={profile.mobile}
+                  style={{
+                    borderColor: errors.mobile ? theme.colors.red : theme.colors.lightGray,
+                    borderRadius: 24,
+                    borderWidth: 1,
+                    color: errors.mobile ? theme.colors.red : theme.colors.text,
+                    padding: 10,
+                    position: 'relative',
+                  }}
+                  disableBorder
                 />
+                <Section.Row style={styles.suffixIcon}>
+                  <Icon
+                    color={errors.mobile ? theme.colors.red : theme.colors.primary}
+                    name="phone"
+                    size={28}
+                    style={styles.phoneIcon}
+                  />
+                </Section.Row>
               </Section.Row>
-            </Section.Row>
-            <ErrorText error={errors.mobile} style={styles.errorMargin} />
-          </Section.Stack>
-        ) : (
+              <ErrorText error={errors.mobile} style={styles.errorMargin} />
+            </Section.Stack>
+          ) : (
+            <InputRounded
+              disabled={true}
+              error={errors.mobile}
+              icon="phone"
+              iconColor={theme.colors.primary}
+              iconSize={28}
+              placeholder="Add your Mobile"
+              value={profile.mobile}
+            />
+          )}
+        </Section.Row>
+        <Section.Row style={!editable && styles.borderedBottomStyle}>
           <InputRounded
-            disabled={true}
-            error={errors.mobile}
-            icon="phone"
+            disabled={!editable}
+            error={errors.email}
+            icon="envelope"
             iconColor={theme.colors.primary}
-            iconSize={28}
-            placeholder="Add your Mobile"
-            value={profile.mobile}
+            iconSize={20}
```

Comment:
why is icon size changed? how does it not modify the current look and feel?

Reply:
The netlify version icon size is 20 on the contrary the email icon is more big than others.
![Screen Shot 2020-02-05 at 11 03 02 AM](https://user-images.githubusercontent.com/59056508/73848587-28850a80-4807-11ea-8c2e-ce1d2a49a698.png)